### PR TITLE
contributors guide (DRAFT)

### LINKED ISSUE #191: Write GitHub Contributor's Guide
Since we want to have a vibrant and active developer community, we should write a GitHub Contributor's Guide so we can help users know how to file issues and PRs against our docs when they notice something that needs to be fixed.

### PR DESCRIPTION
**Pull request by trichards-hiro**
## Description

Added new Hiro Contributor's Guide to Docs site to help developers provide comments and feedback on our content.

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

Fixes #191

### CODE REVIEW

File: docs/contributors-guide.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+To submit a pull request to update the docs, fork the Hiro Docs repository and then clone it.
+
+Currently all Hiro docs are located in `/hirosystems/docs`
+
+1. Create a branch and switch to it:
+    
+    `git checkout -b <branch-name>`
+    
+2. Add or modify the GitHub Markdown files in these directories.
+3. When the changes have been aded, commit them with a message summarizing the changes:
```
Comment (LakshmiLavanyaKasturi) [inline docs/contributors-guide.md:38]: When the changes are added ....

File: docs/contributors-guide.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    
+    `git commit -m "commit message"`
+    
+4. Push the branch up:
```
Comment (LakshmiLavanyaKasturi) [inline docs/contributors-guide.md:42]: Now, push the local branch to the remote repository to create a pull request.

File: docs/contributors-guide.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+title: Hiro Contributor's Guide
+---
+
+# Introduction
+
+One of the most important ways that Hiro can innovate and improve the Stacks developer community and ecosystem is by having contributions to the Hiro product documentation. Having this feedback loop and insight into what works, and what does not work, is helpful in prioritizing what content to write, what content to update, and what content is missing.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: like this much better....fixed. Thank You :)

[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: Fixed

[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: This is better....I went ahead and made this change. Thank You :)

File: docs/contributors-guide.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+This guide describes how to contribute to Hiro docs and content so feedback is received. The documentation team can then incorporate these comments to improve the overall developer documentation experience.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: Fixed

File: docs/contributors-guide.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+To submit a pull request to update the docs, fork the Hiro Docs repository and then clone it.
+
+Currently all Hiro docs are located in `/hirosystems/docs`
+
+1. Create a branch and switch to it:
+    
+    `git checkout -b <branch-name>`
+    
+2. Add or modify the GitHub Markdown files in these directories.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md:37]: Added to document

File: docs/contributors-guide.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+This guide describes how to contribute to Hiro docs and content so feedback is received. The documentation team can then incorporate these comments to improve the overall developer documentation experience.
+
+# **Contributing to hiro documentation**
+
+Hiro welcomes all contributions to Hiro documentation. These contributions come in two forms: issues and pull requests.
+
+## **Issues**
+
+If content is obsolete, technically inaccurate, or unclear, [please create a GitHub Issue](https://github.com/hirosystems/docs/issues/new). This is a great way to give specific feedback and ensures that the content is up-to-date and technically accurate.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: Fixed

File: docs/contributors-guide.md (hunk 1 above)
Comment (trichards-hiro) [reply docs/contributors-guide.md]: Fixed

File: docs/contributors-guide.md (hunk 2 above)
Comment (trichards-hiro) [reply docs/contributors-guide.md]: This is much better.....thank you :)

File: docs/contributors-guide.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    `git push origin <branch-name>`
+    
+
+When all changes are finalized, create a pull request from the branch by following [GitHub's guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+
+### **Titles and descriptions**
+
+Pull request titles should be descriptive enough for reviewers to understand *what* is being changed. Some ways of doing this are better than others.
+
+make sure every pull request has a description that explains *why* the change is being made. The description adds context that is critical for reviewers when giving feedback.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: fixed

File: docs/contributors-guide.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+- For quick GitHub refresher, please refer to the [GitHub Get Started Guide](https://docs.github.com/en/get-started/using-git/about-git).
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply docs/contributors-guide.md]: Fixed

**Pull request closed without merging**